plugins/tmux-autoattach/tmux-autoattach.plugin.sh

```diff
@@ -16,11 +16,11 @@
 # [1] https://github.com/ohmybash/oh-my-bash/pull/332
 
 function _osh_plugin_tmux_autoattach_exit {
-  [[ ! $TMUX ]] && tmux -2u new -As0 && exit
+  [[ -z ${TMUX-} ]] && tmux -2u new -As0 && exit
```

Comment:
`!` is preferred to `-z`. Please revert this. Actually, I want you to use `!` also in `zellij-autoattach`.

Reply:
So replace all `-z` with `!` ?

Reply:
Yes.